_pages/en_US/dumping-cartridges.md

```diff
@@ -5,23 +5,26 @@ redirect_from:
   - /dump-card
 ---
 
-This is an add-on section for dumping cartridges using GodMode9i, useful for loading the cartridge on emulators, flashcarts or your SD card via nds-bootstrap.
+This section is for dumping cartridges using GodMode9i, so they can be played on emulators, flashcarts or your SD card via nds-bootstrap.
```

Comment:
The first part of this sentence feels weird.

Reply:
```suggestion
This section is for dumping cartridges using GodMode9i so they can be played on emulators, flashcarts or your SD card via nds-bootstrap.
```

I think it'd sound better without this comma, do you think that looks good?

Reply:
Yep, sounds better now. Ironic how just a simple character could change an entire viewpoint in a sentence.